### PR TITLE
Add experimental Dockerfile

### PR DESCRIPTION
**Pull request by jolexa**
Hello,

Thanks for your talk at Monitorama, @jrsquared 

I put this example in a Dockerfile to quickly demo to other people not at the conf, maybe it is useful to include here as well? I won't be offended if this doesn't get merged.

If you want to try out the image, feel free to run `jolexa/vizceral-example` (https://hub.docker.com/r/jolexa/vizceral-example/)

### CODE REVIEW

**Comment by adomeier** [comment]
Nice way to be able to stand up quickly for an interactive POC.

File: README.md [hunk 1]
```diff
@@ -14,4 +14,16 @@ For more details about using vizceral in your own projects with your own data, r
...
 
-2. Open `localhost:8080` in your browser.
\ No newline at end of file
+2. Open `localhost:8080` in your browser.
+
+##### Using Docker
+If you don't have a node environment setup or would like to run this example on a platform, there is a Dockerfile for experimental usage.
+
+```
+$ docker build -t <name>/vizceral-example
```
Comment (jrsquared) [inline README.md]: I needed to run `docker build -t <name>/vizceral-example .` (note the trailing dot).

**Comment by jrsquared** [comment]
Looks great! Thanks for doing this.  Just one comment on the readme.

**Comment by jolexa** [comment]
Of course, fixed!

**Comment by jrsquared** [comment]
Merged.  Thanks again!

**Pull request merged**